charms/knative-serving/src/lightkube_helpers/__init__.py

```diff
@@ -0,0 +1,88 @@
+from typing import List, Union, Tuple
```

Comment:
I feel like we are reimplementing lightkube helpers that we already had implemented in the istio-operators repository. Lets start the lightkube library now and consolidate on 1 approach.

Reply:
yes, the intent is for all of this to go in the same place.